package.json

```diff
@@ -26,7 +26,6 @@
   "devDependencies": {
     "@types/jest": "^27.0.2",
     "codecov": "^3.8.3",
-    "got": ">=11.8.5",
```

Comment:
the PR title says revert, but you remove the library entirely? I assume we don't use it at all? It's add that it has the `>=` range limitation, I usually wouldn't use that

Reply:
@gr2m I added that a while ago. I though that it is the cause of the deployment issue but it turns out it is no 😅